stix_shifter_modules/qradar/tests/stix_translation/qradar_stix_to_aql/test_qradar_events_stix_to_query.py

```diff
@@ -439,3 +439,17 @@ def test_inoffense_query(self):
         query = _translate_query(stix_pattern)
         where_statement = "WHERE INOFFENSE('125') {} {}".format(default_limit, default_time)
         _test_query_assertions(query, selections, from_statement, where_statement)
+
+    def test_unmapped_operator_in_combined_comparison(self):
+        stix_pattern = "[ipv4-addr:value ISSUPERSET '127.0.0.1/24' OR ipv4-addr:value ISSUBSET '127.0.0.1/24']"
+        query = _translate_query(stix_pattern)
+        where_statement = "WHERE (INCIDR('127.0.0.1/24',sourceip) OR INCIDR('127.0.0.1/24',destinationip) OR INCIDR('127.0.0.1/24',identityip)) {} {}".format(default_limit, default_time)
+        _test_query_assertions(query, selections, from_statement, where_statement)
+    
+    def test_unmapped_operators_error(self):
+        stix_pattern = "[ipv4-addr:value ISSUPERSET '127.0.0.1/24']"
+        query = _translate_query(stix_pattern)
+        assert query['success'] == False
+        assert ErrorCode.TRANSLATION_MAPPING_ERROR.value == query['code']
+        assert "data mapping error : Unable to map the following STIX objects and properties: " \
+            "['ipv4-addr:value'] or Operators: [IsSuperSet] to data source fields" in query['error']
```

Comment:
Showing both of these kind of masks what is actually causing the error. I know in this case that it's the operator, but it might be good to split the errors up, or put some conditions in the error code to choose what gets printed out.

Reply:
yeah I thought about it. But my intention was showing the users not only the unmapped operators but also the objects that are associated with the error. if this is not the correct approach then sure I can separate the errors.

Reply:
I see your point. You want to show the context where that operator is used. Maybe have three conditions for the error message: 1. If you have both unmapped fields and operators, you show what you have here, just replace the wording to read "and" Operators. 2. If you have just unmapped fields, remove any mention of operators. 3. If you have just unmapped operators, remove mention of unmapped objects and properties, but still show the errored query.

Reply:
updated the error messages